webapp/src/components/groups_input/groups_input.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import debounce from 'lodash/debounce';
+import AsyncSelect from 'react-select/async';
+
+export default class GroupsInput extends React.Component {
```

Comment:
We don't use class based components anymore in main product but I'm not going to block the PR over it

Reply:
I copied what we used for the `UsersInput` component, and made minimal changes. Didn't want to re-invent the wheel, and keep the PR easy to review.

I can swap these to use class components, but would that be better to do in a future PR? I have a feeling we have other class components that also would need to be converted.

Reply:
Yeah I think that's something we do another time, fine to leave as a class component for now